api/policy.go

```diff
@@ -32,6 +37,11 @@ const (
 	// PipelineApprovalValueManualAlways means the pipeline should be manually approved by user to proceed.
 	PipelineApprovalValueManualAlways PipelineApprovalValue = "MANUAL_APPROVAL_ALWAYS"
```

Comment:
What if we use approval policy,  add MANUAL_APPROVAL_ALWAYS_PROJECT_OWNER_ONLY

Reply:
+1 for this approach.
It adds a new value instead of a new type.

Reply:
SGTM